AntistasiOfficial.Altis/Missions/SUP_CitySupply.sqf

```diff
@@ -0,0 +1,227 @@
+private ["_targetPosition","_targetName","_duration","_endTime","_task", "_posSupply", "_FIAMarkers", "_nFIAMarker","_spawnPosition","_crate", "_house", "_range","_allBuildings","_groupType","_params","_group","_dog","_leaflets","_drop"];
+
+
+_tskTitle = "City Supplies";
+_tskDesc = "Get it from the AAF and deliver it to %1 before %2:%3";
+_tskDesc_drop = "Now bring this supplies to %1 before %2:%3";
+_tskDesc_fail = "You failed delivering a paket to %1, very impressive";
+_tskDesc_success = "You crazy little postman";
+
+_targetPosition = getMarkerPos _marker;
+_targetName = [_marker] call AS_fnc_localizar;
+_range = [_marker] call sizeMarker;
+_posHQ = getMarkerPos guer_respawn;
+
+/*
+_allBuildings = nearestObjects [_targetPosition, ["Building"], _range];
+if (count _allBuildings == 0) exitWith {};
+_targetBuilding = selectRandom _allBuildings; <= if house as dropoff point wanted use this
+*/
+
+_targetBuilding = _targetPosition;
+
+_duration = 30;
+_endTime = [date select 0, date select 1, date select 2, date select 3, (date select 4) + _duration];
+_endTime = dateToNumber _endTime;
+/*
+_FIAMarkers = mrkFIA + campsFIA;
+
+while {true} do {
+	sleep 0.1;
+	_posSupply = [_targetPosition,2000,random 360] call BIS_fnc_relPos;
+	_nFIAMarker = [_FIAMarkers,_posSupply] call BIS_fnc_nearestPosition;
+	if ((!surfaceIsWater _posSupply) && (_posSupply distance _posHQ < 4000) && (getMarkerPos _nFIAMarker distance _posSupply > 500)) exitWith {};
+};
+*/
+_houseType = "Land_i_Shed_Ind_F";
+_crateType = "Land_PaperBox_01_open_boxes_F";
+if(_type == "WATER") then
+{
+	_crateType = "Land_PaperBox_01_open_water_F";
+};
+if(_type == "FUEL") then
+{
+	_crateType = "CargoNet_01_barrels_F";
+};
+_allSheds = nearestObjects [_targetPosition, [_houseType], 2000, true];
+if (count _allSheds == 0) exitWith
+{
+	diag_log format ["Supply mission not created, could not find %1 around %2", _houseType, _targetPosition];
+};
+
+_spawnPosition = _allSheds selectRandom;
+_spawnPosition = _spawnPosition findEmptyPosition [5,50, _crateType];
+sleep 1;
+_crate = _crateType createVehicle _spawnPosition;
+_crate allowDamage false;
+[_crate] spawn {sleep 1; (_this select 0) allowDamage true;};
+_crate call jn_fnc_logistics_addAction;
+//Spawned the crate in
+
+/*
+_marker = createMarker [format ["REC%1", random 100], _posSupply];
+_marker setMarkerShape "ICON";
+
+_task = ["SUP",[side_blue,civilian],[[_tskDesc,_targetName,numberToDate [2035,_endTime] select 3,numberToDate [2035,_endTime] select 4],_tskTitle,_marker],_posSupply,"CREATED",5,true,true,"Heal"] call BIS_fnc_setTask;
+
+*/
+
+_allGroups = [];
+_allSoldiers = [];
+
+_groupType = [infSquad, side_green] call AS_fnc_pickGroup;
+_group = [_posSupply, side_green, _groupType] call BIS_Fnc_spawnGroup;
+_allGroups pushBack _group;
+
+{
+	_group = _x;
+	{
+		[_x] spawn genInit; //Or genInitBASES ?? Where is the difference between them?
```

Comment:
@StefArma Is there a difference between genInit and genInitBASES?

Reply:
yes, genInitBases is for Base and Airport

Reply:
Ok no other difference?